DataFormats/TrackerRecHit2D/interface/MTDTrackingRecHit.h

```diff
@@ -0,0 +1,42 @@
+#ifndef DataFormats_TrackerRecHit2D_MTDTrackingRecHit_h
+#define DataFormats_TrackerRecHit2D_MTDTrackingRecHit_h
+
+/// A 2D TrackerRecHit with time and time error information
+
+#include <cassert>
+#include "DataFormats/TrackerRecHit2D/interface/TrackerSingleRecHit.h"
+#include "DataFormats/FTLRecHit/interface/FTLRecHitCollections.h"
+
+class MTDTrackingRecHit : public TrackerSingleRecHit {
+    public:
+        
+        MTDTrackingRecHit() : TrackerSingleRecHit() {}
+        
+	MTDTrackingRecHit(const LocalPoint& p, const LocalError& e,
```

Comment:
inconsistent indentation

Reply:
This part has been removed from this PR.